src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemProvider.cs

```diff
@@ -112,7 +113,18 @@ private void StartSearch(INavigateToCallback callback, string searchValue, IImmu
                 kinds,
                 _threadingContext.DisposalToken);
 
-            _ = searcher.SearchAsync(searchCurrentDocument, _cancellationTokenSource.Token);
+            _ = SearchAsync(searcher, searchCurrentDocument, _cancellationTokenSource.Token);
+        }
+
+        private async Task SearchAsync(NavigateToSearcher searcher, bool searchCurrentDocument, CancellationToken cancellationToken)
+        {
+            try
+            {
+                await searcher.SearchAsync(searchCurrentDocument, cancellationToken).ConfigureAwait(false);
+            }
+            catch (OperationCanceledException)
+            {
+            }
```

Comment:
we still need a `catch(Exception) FatalError...` otherwise true failures will be lost into the ether.